Lib/inspect.py

```diff
@@ -2971,6 +2973,20 @@ def __init__(self, parameters=None, *, return_annotation=_empty,
                             # There is a default for this parameter.
                             seen_default = True
 
+                    if kind == _VAR_POSITIONAL:
+                        if seen_var_positional:
+                            msg = 'more than one var-positional parameter'
+                            raise ValueError(msg)
```

Comment:
```suggestion
                            raise ValueError(
                            	'more than one var-positional parameter')
```

If it exceeds 80 chars.